spec/presenters/redetermination_presenter_spec.rb

```diff
@@ -1,19 +1,20 @@
 require 'rails_helper'
 
 describe RedeterminationPresenter do
-
-  let(:claim)           { FactoryBot.create :claim, apply_vat: true }
-  let(:rd)              { FactoryBot.create :redetermination, fees: 1452.33, expenses: 2455.77, disbursements: 2123.55, claim: claim }
-  let(:presenter)       { RedeterminationPresenter.new(rd, view) }
+  let(:claim) { create :claim, apply_vat: true }
+  let(:rd) { create :redetermination, fees: 1452.33, expenses: 2455.77, disbursements: 2123.55, claim: claim }
+  let(:presenter) { RedeterminationPresenter.new(rd, view) }
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2271/files#r173431420)

Reply:
👍